### PR TITLE
Fix device run command

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
When executing `appbuilder device run appid` outside of project the CLI cannot determine which startPackageActivity name to use - for NativeScript or for Cordova application. The fix is to check where the command is executed and if it is not in project we need to run the adb command for both NativeScript and Cordova projects.

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:broken_heart:

**Comment by justcodebuilduser** [comment]
:broken_heart:

**Comment by justcodebuilduser** [comment]
:heart:

File: mobile/mobile-core/devices-service.ts [hunk 1]
```diff
@@ -522,6 +523,11 @@ export class DevicesService implements Mobile.IDevicesService {
 	private clearCurrentDeviceDetectionIntervalFuture(): IFuture<void> {
 		return this.deviceDetectionIntervalFuture || Future.fromResult();
 	}
+
+	private getStartPackageActivity(framework?: string): string {
```
Comment (Mitko-Kerezov) [inline mobile/mobile-core/devices-service.ts]: This method appears to be Android-specific, you can consider moving it to some Android-specific class, like for example `AndroidApplicationManager`

**Comment by Mitko-Kerezov** [comment]
👍 after either moving `getStartPackageActivity` method's location or giving a reasonable explanation not to

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:heart:

File: mobile/mobile-core/devices-service.ts [hunk 2]
```diff
@@ -3,8 +3,8 @@ import Future = require("fibers/future");
 import * as helpers from "../../helpers";
 import * as assert from "assert";
 import * as constants from "../constants";
-import {exportedPromise, exported} from "../../decorators";
 import * as fiberBootstrap from "../../fiber-bootstrap";
+import {exportedPromise, exported} from "../../decorators";
```
Comment (rosen-vladimirov) [inline mobile/mobile-core/devices-service.ts]: why is this changed ?

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by justcodebuilduser** [comment]
:heart:
